modules/mutes.ts

```diff
@@ -3,20 +3,20 @@ import { Client, Guild, GuildMember, Role, Message } from 'discord.js'
 const getMutesRole = async (message: Message) => {
   let muteRole = await message.client['guildData'].get(`${message.guild.id}.mutedRole`)
   if (muteRole == null) {
-    muteRole = message.guild.roles.find(r => r.name === 'mutes')
+    muteRole = message.guild.roles.find(r => r.name === 'muted')
   }
   return muteRole
 }
 
-export const checkMutes = async (client: Client) => {
-  let mutedRole = async (guild: Guild): Promise<Role> => {
-    let role = client['guildData'].get(`${guild.id}.mutedRole`)
-    if (role == null) {
-      role = guild.roles.find(x => x.name === 'mutes')
-    }
-    return role
+const getGuildMutesRole = async (guild: Guild): Promise<Role> => {
+  let role = client['guildData'].get(`${guild.id}.mutedRole`)
```

Comment:
How does this method access `client` now?

Reply:
oh, it was meant to be guild.client